backend/database/Messages.ts

```diff
@@ -88,7 +88,10 @@ class DatabaseMessageUtils {
       if (lineCountExists.length < 1) {
         await knex('line_counts').insert({ count: 0, botLines: 1, date: this.formatDate(new Date()) });
```

Comment:
Shouldn't the `count` be increased here as well? As it's done below as well.

Reply:
made an emergency commit after merging. didn't catch this in time for PR but master has this updated correctly.